packages/server/server.ts

```diff
@@ -2,17 +2,15 @@ import express, { Express, Request, Response, json } from "express";
 import https from "https";
 import fs from "fs";
 import cors from "cors";
-import dotenv from "dotenv";
 import {
   validateSubmission,
   fetchChallenge,
   downloadContract,
   testChallengeSubmission,
 } from "./utils";
+import { PORT } from "./utils";
```

Comment:
This is nitpicky but you could include the `PORT` import with the other `./utils` imports

Reply:
Haha Thanks for pointing that out. It bugs me that I missed that.